sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -244,6 +244,9 @@ statement
     | SET .*?                                                          #setConfiguration
     | RESET configKey                                                  #resetQuotedConfiguration
     | RESET .*?                                                        #resetConfiguration
+    | CREATE indexType=STRING? INDEX (IF NOT EXISTS)? identifier ON TABLE?
```

Comment:
```suggestion
    | CREATE indexType=identifier? INDEX (IF NOT EXISTS)? identifier ON TABLE?
```

STRING only matches `'abc'`, but we want to support `CREATE bloom_filter INDEX ...`